src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService+UITestingMethods.swift

```diff
@@ -170,5 +157,43 @@ extension HealthCertificateService {
 			}
 		}
 	}
+	
+	func updateDccWalletInfoForMockBoosterNotification(dccWalletInfo: DCCWalletInfo) -> DCCWalletInfo {
+		let titleText = DCCUIText(
+			type: "string",
+			quantity: nil,
+			quantityParameterIndex: nil,
+			functionName: nil,
+			localizedText: ["de": "Hinweis zur Auffrischimpfung"],
+			parameters: []
+		)
```

Comment:
We have a more compact `fake(string:)` that would make this shorter

Reply:
Those fake functions are only available in target ENATests. Since this function is an extension of HealthCertificateService it's an ENA member.